components/card/card.html

```diff
@@ -0,0 +1,223 @@
+                        <li class="link"><a href="">Library</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </div>
+        <div class="document-container">
+            <div class="sidebar">
+                <div class="doc-topic">
+                    Documentation
+                </div>
+                
+                <ul class="doc-list">
+                    <li class="doc-item">Introduction</li>
+                    <li class="doc-item">Colors</li>
+                    <li class="doc-item">Text Utilities</li>
+                </ul>
+                <hr>
+                <div class="components">
+                    Components
+                </div>
+                
+                <ul>
+                    <li class="comp-name">Alert</li>
+                    <li class="comp-name">Avatar</li>
+                    <li class="comp-name">Badge</li>
+                    <li class="comp-name">Button</li>
+                    <li class="comp-name">Card</li>
+                    <li class="comp-name">Image</li>
+                    <li class="comp-name">Input</li>
+                    <li class="comp-name">Lists</li>
+                    <li class="comp-name">Navigation</li>
+                    <li class="comp-name">Snackbar</li>
+                </ul>
+
+            </div>
+            <div class="main-document">
+                <div class="card-head">
+                    Card
+                </div>
+                <div class="card-content">
+                    Tec{h}tonic gives you some types of card for different uses in your website.
+                </div>
+                <hr>
+                <div class="card-type">
+                    Card with badges
+                </div>
+                <div class="card-container">
+                    <article class="complete-card relative">
+                        <img src="../../assets/responsive-image.jpg" alt="" class="card-image">
+                        <div class="main-info-cont">
+                            <h3 class="info-head">Cosmos</h3>
+                            <p class="sm-txt">Our Earth</p>
+                        </div>
+                        <div class="sub-info-cont">
+                            <p class="sub-info">The Stardust we are floating through.</p>
+                        </div>
+                        <div class="footer">
+                            <button class="button read-btn">Read</button>
+                            <button class="button bookmark-btn">Bookmark</button>
+                            <div class="icon-cont">
+                                <button class="card-icon"><i class="fas fa-heart"></i></button>
+                                <button class="card-icon"><i class="fas fa-share-alt"></i></button>
+                                <button class="card-icon"><i class="fas fa-ellipsis-v"></i></button>
+                            </div>
+                        </div>
```

Comment:
Don't use footer, header as class names. They are semantic elements on their own.  Since this is something within a card there could have been better choice of class name such as `card-links-container`

Reply:
Okay, I will keep this in mind and will not use it.